x/ibc-rate-limit/ibc_middleware_test.go

```diff
@@ -506,6 +506,6 @@ func (suite *MiddlewareTestSuite) TestUnsetRateLimitingContract() {
 	osmosisApp := suite.chainA.GetOsmosisApp()
 	paramSpace, ok := osmosisApp.AppKeepers.ParamsKeeper.GetSubspace(types.ModuleName)
 	suite.Require().True(ok)
-        // N.B.: this panics if validation fails.
+	// N.B.: this panics if validation fails.
```

Comment:
Can't comment on that file but please ever the rate limiter wasm

Reply:
Sorry I don't quite understand what you mean here

Reply:
> Sorry I don't quite understand what you mean here

Please remove `rate_limiter.wasm`

What I meant is that I could not comment under that file so the comment location is wrong / confusing